scripts/ci/docker-compose/backend-mssql-docker-volume.yml

```diff
@@ -20,3 +20,5 @@ services:
   mssql:
     volumes:
       - mssql-db-volume:/var/opt/mssql
+volumes:
```

Comment:
I added some extra code for "docker stop" command to include all backend compose files, so we can now move the volumes finally to where they belong.